### PR TITLE
Testing user

### PR DESCRIPTION
**Pull request by emmadavids**

### CODE REVIEW

File: spec/models/user.spec.js [hunk 1]
```diff
@@ -1,58 +1,165 @@
...
+      username: "existinguser2",
+    });
+  
+  
+    await existingUser1.save();
+ 
+    try {
+      await existingUser2.save();
+    } catch (error) {
+      expect(error.errors["email"].message).toBe("Email address is already in use");
```
Comment (adrianHards) [inline spec/models/user.spec.js:40]: not sure any of these catch tests are being wrong

**Review by adrianHards — CHANGES_REQUESTED** [review]
not sure any of these catch tests are being wrong

File: spec/models/user.spec.js (hunk 1 above)
Comment (emmadavids) [reply spec/models/user.spec.js]: I've pushed again if that's showing ? Still have 2 failing tests but wonder if someone else wouldnt mind taking a look as my head hurts.

**Comment by emmadavids** [comment]
still have 2 failing tests. I wonder if someone else could take a look at them

**Review by adrianHards — APPROVED** [review]
Nice

**Pull request merged**